### DISCUSSION TITLE
[V5] The new @loggedIn ?

### DISCUSSION THREAD

**Discussion by Reptiluka** [adonisjs/core#1841] [Help] [answered]
Hi !
I found the @loggedIn tag from V4, but dont works in V5
What I should do for the equivalent ? ex. Display or not a section for an auth user in the page

**Comment by thetutlage** [comment, marked as answer]
You can use the `auth` object directly.

```edge
@if(auth.user)
  <p> User is logged in </p>
@endif
```

**Reply by Reptiluka** [reply]
I tried this first, but didnt work the first time,
Should I return the auth object in each of my view ?

**Reply by Reptiluka** [reply]
Even if I return auth it didnt work.. I dont understand...

**Reply by RomainLanz** [reply]
Since this is the correct answer, let me add a side note.

If you want to use this in all your template, meaning also routes that doesn't have the `auth` middleware, you need to activate the `SilentAuth` middleware.

The documentation cover this topic, feel free to check the following link.
▶ https://preview.adonisjs.com/guides/auth/middleware#silent-auth-middleware

**Comment by Reptiluka** [comment]
@thetutlage Also auth.user is only accessible when I add the auth middleware,
with the silent, im able to access ctx.auth.user , but only in Route.ts , I'm not able to access ctx in the controller... and ever worse in the view....

The doc looks unclear and we should always suppose that thing and that thing and add this .... whyyyyy ?

**Comment by Reptiluka** [comment]
EDIT : Don't know if it was because i added silentauth or not, but not i can access auth object in view, finally...

Maybe it was a cache issue from the server ? I tried to restart and didnt change, but 6 hours later, finally works !
